apps/renderer/src/modules/feed-column/corner-player.tsx

```diff
@@ -222,20 +227,21 @@ const PlayerProgress = () => {
     setControlledCurrentTime(currentTime)
   }, [currentTime, isDraggingProgress])
 
-  const currentTimeIndicator = dayjs()
-    .startOf("y")
-    .second(controlledCurrentTime)
-    .format(controlledCurrentTime > ONE_HOUR_IN_SECONDS ? "H:mm:ss" : "m:ss")
+  const getTimeIndicator = useCallback((time: number) => {
```

Comment:
```suggestion
  const getTimeIndicator =((time: number) => {
```

`useCallback` is unnecessary